### DISCUSSION TITLE
Interactive API Docs Shows a Blank Page

### DISCUSSION THREAD

**Discussion by rachari** [fastapi/fastapi#7013] [Questions] [answered]
I was trying the sample mentioned on the https://fastapi.tiangolo.com/#installation page.

```fastapi           0.63.0```
```Python 3.8.5```

The API calls work as required via curl / browser, however the Interactive API docs / Swagger UI doesn't load.

Checking with Chrome DevTools shows the following :

```GET https://cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui.css net::ERR_CERT_AUTHORITY_INVALID```
```GET https://cdn.jsdelivr.net/npm/swagger-ui-dist@3/swagger-ui-bundle.js net::ERR_CERT_AUTHORITY_INVALID```
```docs:15 Uncaught ReferenceError: SwaggerUIBundle is not defined```

This seems to be an issue with the CDN from where the Swagger UI related resources are being fetched.

Is there any alternative that can be used for this ?

**Comment by Kludex** [comment, marked as answer]
Duplicated? https://github.com/tiangolo/fastapi/issues/2735

**Comment by rachari** [comment]
My bad, yes, this duplicates issue #2735

Marking this as closed.

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex ! 👏  🙇 

Thanks for reporting back and closing the issue @rachari  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
